docs/tutorials/open-source/apache-flink/0_setup.qmd

```diff
@@ -0,0 +1,35 @@
+# Getting started
+
+## Overview
+
+[Ibis](https://ibis-project.org/) is a Python library for writing and scaling data transformations. With close to 20 supported backends today, it allows you to write transformations once and reuse the same code for execution across multiple backends (e.g. pandas, Postgres, etc).
+
+In today’s data infrastructures, it’s common for data scientists to do development locally and experiment with a sample dataset in Python. If the results look promising, the features and models would then be deployed into production. Oftentimes, there is a code handover in this process and a dedicated team of developers would be responsible for rewriting the logic for production. This incurs a huge amount of overhead.
+
+With the launch of support for the Flink backend in Ibis, we can now reuse the same code to work directly with streaming data in production. You can connect to data sources and sinks and write transformations in Python for both local experimentation and production deployment. Ibis provides a dataframe API that data scientists should already be familiar with, so that you don’t have to spend time learning a new query/processing language.
```

Comment:
also would rephrase more simply -- e.g. in the Starburst Galaxy tutorial we just have: "In this tutorial, we will connect to Starburst Galaxy and verify our connection. Following tutorials will go through the basics of Ibis on Starburst Galaxy’s demo data."